### PR TITLE
Hw02 express

### PR DESCRIPTION
**Pull request by ArtemOsypov**
Dear Mentor,
Please check my HW#2

### CODE REVIEW

File: routes/api/contacts-router.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+import { HttpError } from "../../helpers/index.js";
+
+import Joi from "joi";
+
+const contactsRouter = express.Router();
+
+const contactAddSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().required(),
```
Comment (ceREAL133) [inline routes/api/contacts-router.js]: email and phone should be validated properly. Schema should be in separate file

File: routes/api/contacts-router.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  phone: Joi.string().required(),
+});
+
+contactsRouter.get("/", async (req, res, next) => {
+  try {
+    const result = await contactsService.listContacts();
```
Comment (ceREAL133) [inline routes/api/contacts-router.js]: functions in routes should be in separate file. THey must just be invoked here

**Review by ceREAL133 — CHANGES_REQUESTED** [review]

File: routes/api/contacts-router.js (hunk 1 above)
Comment (ArtemOsypov) [reply routes/api/contacts-router.js]: Done

File: routes/api/contacts-router.js (hunk 2 above)
Comment (ArtemOsypov) [reply routes/api/contacts-router.js]: Done

**Review by ceREAL133 — APPROVED** [review]

**Pull request merged**